### ISSUE TITLE
Problema datos contrato diferente titular y pagador

### ISSUE THREAD

**Issue by bpesquero** [PKFEric/webProxima#420]
Para el contrato COM/00995 el cliente en el portal web ve un cruce de datos relativos a la dirección fiscal al acceder como titular.

NAVISION:

-         Dirección fiscal titular CL00786 /envío: Carrer Panadella,5 08293

·         Dirección Fiscal pagador CL00887/ envío: C/Orenci Valls 7 4º B 08292

·         Dirección suministro: C/Orenci Valls 7 4º B 08292

Sin embargo, en la web, aparece lo siguiente (como si hubiera cruzado el número de la fiscal del titular con la envío).

 
![image](https://user-images.githubusercontent.com/20299071/48475067-45957e00-e7fc-11e8-98c7-0ed8f1626ce3.png)

No sé si es algo que hemos hecho nosotros mal, porque este contrato se dio de alta a mano. PEro la verdad es que no lo veo.

**Comment by jlagunapkf** [comment]
En el contrato, ¿la dirección fiscal es la del pagador o la del titular?

**Comment by bpesquero** [comment]
La del titular

**Comment by JMunoz17** [comment]
En el contrato la dirección se rellena a partir del cliente pagador, entiendo que lo correcto es que lo haga a partir de la del titular no.

Lo modificamos y pasamos un proceso para arreglarlo en todos los contratos ok?

**Comment by bpesquero** [comment]
Si. Esto también pasa en A tu lado?
Habria que solucionarlo en los dos. 

En cualquier caso te das cuentas que no todos los datos de la dirección los coge del pagador. Mezcla, la dirección la coge de un sitio y el CP y la población de otro.

**Comment by JMunoz17** [comment]
Si, esto aplica tanto a GEO como a HECEN. Lo modificaremos para que la dirección la obtenga del titular en lugar del pagador como hasta ahora y pasamos un proceso para ambas empresas.

Te indico que datos se obtienen del pagador y cuales del titular por si ves alguno mas que no este bien:
**Pagador:**
Nombre cliente
Población
Nº Teléfono
Nº Telex
C.P.
Provincia
Cód.País Región
Dirección de envío y todos los datos asociados a esta

**Titular**
Nombre Titular
Nombre 
Primer Apellido
Segundo Apellido
Piso, portal, escalera, número

**Comment by bpesquero** [comment]
PErdona, pero no entiendo muy bien.

te refieres al pantallazo?
En el pantallazo no hay titular y cliente... No estoy entendiendo lo que me dices.

**Comment by JMunoz17** [comment]
No, me refiero a los datos en el contrato. Aunque no se muestren en la web, alomejor hay alguno que no se esta llevando del cliente correcto (es el problema que hemos tenido con la dirección, la cual obtenemos del contrato), por eso te preguntaba por si hay alguno mas que no sea correcto.

**Comment by bpesquero** [comment]
Yo creo que para aclarararnos los mejor es coger un contrato, una pantallazo, y que me pongas al lado de donde se coge cada dato, porque me estoy liando.

Y lo mismo con el detalle del contrato que aparece en la web

Te parece?

**Comment by JMunoz17** [comment]
Contestado vía email.

**Comment by JMunoz17** [comment]
Hemos realizado las modificaciones acordadas y pasado el proceso en GEO y en HECEN.
